lib/evmone/eof.cpp

```diff
@@ -0,0 +1,54 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "eof.hpp"
+
+#include <array>
+
+namespace evmone
+{
+namespace
+{
+constexpr uint8_t FORMAT = 0xef;
+constexpr uint8_t MAGIC[] = {0xa6, 0x1c};
```

Comment:
Why not use the same as in geth/tests?

Reply:
I took this from #303 

Yeah we should better change it to have the same thing everywhere.